gh2/csv.py

```diff
@@ -167,13 +175,16 @@ def normalize_sequential_dates(issue_list):
 
 
 def write_rows(filename, headers, fields, issues, date_format, include_prs,
-               skip_normalization, additional_labels):
+               skip_normalization, additional_labels, filter_labels=None):
     with open(filename, 'w') as fd:
         writer = csv.writer(fd)
         writer.writerow(headers)
         for issue in issues:
             if not include_prs and is_pull_request(issue):
                 continue
+            issue_labels = [l.name for l in issue.labels()]
```

Comment:
Given this is used as a set, this could be rewritten as:
issue_labels = {l.name for l in issue.labels()}